kernel/fde/mapper.go

```diff
@@ -0,0 +1,49 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2021  Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package fde
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/snapcore/snapd/osutil/disks"
+)
+
```

Comment:
I suppose the function to make "*-device-locked" name will also land here?

Reply:
which function is that? is that from mvo's full wip branch?

Reply:
yes

Reply:
yes that would make sense but I think that specific function should be moved in a different PR here

Reply:
that should have read "that specific function should be moved to this file in a different PR"